chart/otomi/values.yaml

```diff
@@ -1,38 +1,40 @@
 cluster:
-  domainSuffix: ''
+  # domainSuffix: ''
   k8sVersion: '1.20'
   name: 'dev'
   owner: ''
   provider: '' # provider can be one of aws|azure|google
 otomi:
   adminPassword: ''
+  hasExternalDNS: false
   # Set this to true, when you bring your own IDP such as Azure AD. (Then you must also fill in the 'oidc:' settings below.)
   # hasExternalIDP: true
   # release versions can be found here: https://github.com/redkubes/otomi-core/releases
   # by default the image tag is set to .Chart.AppVersion
   # version: master
-dns:
-  provider:
-    # provide one of the following below: aws|azure|google
-    # aws:
-    #   # next two keys are optional for explicit access with an iam role
-    #   # (if no metadata exists with implicit role access to manage dns)
-    #   accessKeySecret: ''
-    #   secretAccessKey: ''
-    #   # region is always needed
-    #   region: eu-central-1
-    #   role: '' # optional ARN, may be set explicitly if no metadata can be accessed
-    # azure:
```

Comment:
I notice in this file we ask the user/dev to provide `provider` for (at least) the following object in our `values.yaml` file: (`cluster.provider`, `dns.provider`, `kms.sops.provider`).

It is very confusing, I am not sure if it is in or out of scope for this PR but what do you think of this?

Reply:
yeah, but what if you have you cluster on Azure and DNS on Google

Reply:
resolving this as we need ppl to be able to choose where they run things